src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import PermissionAlert from 'app/views/settings/project/permissionAlert';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import {fields} from 'app/data/forms/projectGeneralSettings';
+import AsyncView from 'app/views/asyncView';
+import ProjectActions from 'app/actions/projectActions';
+import {Scope} from 'app/types';
+
+import {DataPrivacy} from './types';
+
+class ProjectDataPrivacyContent extends AsyncView<DataPrivacy> {
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
+    router: PropTypes.object,
+  };
+
+  getEndpoints() {
+    const {orgId, projectId} = this.props.params;
+    const endpoints = [['data', `/projects/${orgId}/${projectId}/`]];
+    return endpoints as Array<[string, string]>;
+  }
+
+  renderBody() {
+    const {organization} = this.context;
+    const project = this.state.data;
+    const {orgId, projectId} = this.props.params;
+    const endpoint = `/projects/${orgId}/${projectId}/`;
+    const access = new Set(organization.access);
+    const features = new Set(organization.features);
+    return (
+      <React.Fragment>
+        <SettingsPageHeader title={t('Data Privacy')} />
+        <PermissionAlert />
+        <Form
+          saveOnBlur
+          allowUndo
+          initialData={project}
+          apiMethod="PUT"
+          apiEndpoint={endpoint}
+          onSubmitSuccess={resp => {
+            // This will update our project context
+            ProjectActions.updateSuccess(resp);
+          }}
+        >
+          <JsonForm
+            title={t('Data Privacy')}
+            additionalFieldProps={{
+              organization,
+              groupingConfigs: this.state.groupingConfigs,
+              groupingEnhancementBases: this.state.groupingEnhancementBases,
+            }}
+            features={features}
+            access={(access as unknown) as Array<Scope>}
```

Comment:
Is the access/features prop actually doing anything once it's passed down?

Reply:
you are right about `access`, but regarding the `features`, I cannot remove it as it is being used in places like this:

` visible: ({features}) => features.has('datascrubbers-v2'),`